### PR TITLE
Update README.md with DeepRank2 link

### PR DESCRIPTION
**Pull request by gcroci2**
Any suggestions for improving the description are welcome :D @CunliangGeng

### CODE REVIEW

**Comment by CunliangGeng** [comment]
If it's not recommended to use deeprank any more, I would suggest [archiving this repo](https://docs.github.com/en/repositories/archiving-a-github-repository/archiving-repositories) and give a more clear instruction in the README to direct users to deeprank2.

**Comment by gcroci2** [comment]
> If it's not recommended to use deeprank any more, I would suggest [archiving this repo](https://docs.github.com/en/repositories/archiving-a-github-repository/archiving-repositories) and give a more clear instruction in the README to direct users to deeprank2.

Fine for me, if @LilySnow approves I can do that myself

**Comment by gcroci2** [comment]
@LilySnow has agreed to archive the repo. If @CunliangGeng approves my changes to the readme, I will merge and then I will proceed with the archiving action.

File: README.md [hunk 1]
```diff
@@ -23,6 +37,8 @@ DeepRank is a general, configurable deep learning framework for data mining prot
 
 DeepRank contains useful APIs for pre-processing PPIs data, computing features and targets, as well as training and testing CNN models.
 
+We recently developed [DeepRank2](https://github.com/DeepRank/deeprank2), an upgraded package that includes all the functionalities of DeepRank and more.
```
Comment (CunliangGeng) [inline README.md:40]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -1,3 +1,17 @@
...
+This repository is no longer being maintained and has been archived for historical purposes. 
+
+We have now developed [DeepRank2](https://github.com/DeepRank/deeprank2), an improved and unified version of DeepRank, [DeepRank-GNN](https://github.com/DeepRank/Deeprank-GNN), and [DeepRank-Mut](https://github.com/DeepRank/DeepRank-Mut).
+
+:sparkles: DeepRank2 allows for transformation and storage of 3D representations of both protein-protein interfaces (PPIs) and protein single-residue variants (SRVs) into either graphs or volumetric grids containing structural and physico-chemical information. These can be used for training neural networks for a variety of patterns of interest, using either our pre-implemented training pipeline for graph neural networks (GNNs) or convolutional neural networks (CNNs) or external pipelines.
+
+- :wrench: **Pull Requests** at github.com/DeepRank/deeprank2
+- :bug: **Bugs**: Reports of bugs can be filed agains our new repo github.com/DeepRank/deeprank2
+- :star: **Feature Requests**: Add your request or discuss the project w/ the community at github.com/DeepRank/deeprank2
+
```
Comment (CunliangGeng) [inline README.md:12]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -1,3 +1,17 @@
...
+We look forward to seeing you in our new spaces!
```
Comment (CunliangGeng) [inline README.md:13]:
```suggestion
We look forward to seeing you in our new space - [DeepRank2](https://github.com/DeepRank/deeprank2)!
```

File: README.md (hunk 2 above)
Comment (CunliangGeng) [reply README.md:12]: It's also ok to keep them, but please preview the links and make sure they look good

**Review by CunliangGeng — APPROVED** [review]
Please check my suggested comments. 
After merging this PR, please change the setting of this repo to archive it (see [guide](https://docs.github.com/en/repositories/archiving-a-github-repository/archiving-repositories#archiving-a-repository)).

**Pull request merged**